external/libp2p-service.js

```diff
@@ -51,18 +54,20 @@ class Libp2pService {
                         list: this.config.bootstrapMultiAddress,
                     },
                 };
-            } else {
-                initializationObject.addresses = {
-                    listen: ['/ip4/0.0.0.0/tcp/9000'] // for production
-                    // announce: ['/dns4/auto-relay.libp2p.io/tcp/443/wss/p2p/QmWDn2LY8nannvSWJzruUYoLZ4vV83vfCBwd8DipvdgQc3']
-                };
             }
+            initializationObject.addresses = {
+                listen: [`/ip4/0.0.0.0/tcp/${this.config.port}`] // for production
+                // announce: ['/dns4/auto-relay.libp2p.io/tcp/443/wss/p2p/QmWDn2LY8nannvSWJzruUYoLZ4vV83vfCBwd8DipvdgQc3']
+            };
+
             if (!this.config.peerId) {
-                const configFile = JSON.parse(fs.readFileSync('.origintrail_noderc'));
+                const configFile = JSON.parse(fs.readFileSync(configFilename));
                 if (!configFile.network.privateKey) {
                     const id = await PeerId.create({bits: 1024, keyType: 'RSA'})
                     configFile.network.privateKey = id.toJSON().privKey;
-                    fs.writeFileSync('.origintrail_noderc', JSON.stringify(configFile, null, 2));
+                    if(process.env.NODE_ENV !== 'development') {
+                        fs.writeFileSync('.origintrail_noderc', JSON.stringify(configFile, null, 2));
```

Comment:
Parametrize here name as well

Reply:
Done in [latest commit](https://github.com/OriginTrail/ot-node/pull/1698/commits/8901b8da1462a2047e8df62f1d9e86153edc83eb)